README.md

```diff
@@ -62,8 +62,7 @@ Upon a new component being added to this table run 'npm run update-readme' for b
 [Drawer](https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-drawer--drawer) | <details><summary>Screenshot</summary><img src="src/screenshots/Drawer.png"></details> | [Filter](https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-filter-allfilters--no-filter-bar) | <details><summary>Screenshot</summary><img src="src/screenshots/Filter.png"></details> |
 [AdditionalJSONFilter](https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-filter-additionaljsonfilter--additional-json-filter) | <details><summary>Screenshot</summary><img src="src/screenshots/AdditionalJSONFilter.png"></details> | [DoubleMultiSelectFilter](https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-filter-doublemultiselectfilter--double-multi-select-filter) | <details><summary>Screenshot</summary><img src="src/screenshots/DoubleMultiSelectFilter.png"></details> | 
 [DurationFilter](https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-filter-durationfilter--duration-filter) | <details><summary>Screenshot</summary><img src="src/screenshots/DurationFilter.png"></details>| [ListFilter](https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-filter-listfilter--list-filter) | <details><summary>Screenshot</summary><img src="src/screenshots/ListFilter.png"></details> |
-[MultiSelectFilter](https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-filter-multiselectfilter--multi-select-filter) | <details><summary>Screenshot</summary><img src="src/screenshots/MultiSelectFilter.png"></details> | [RadioFilter](https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-filter-radiofilter--radio-filter) | <details><summary>Screenshot</summary><img src="src/screenshots/RadioFilter.png"></details> | 
-[SingleSelectFilter](https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-filter-singleselectfilter--single-select-filter) | <details><summary>Screenshot</summary><img src="src/screenshots/SingleSelectFilter.png"></details> | [TextFilter](https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-filter-textfilter--text-filter) | <details><summary>Screenshot</summary><img src="src/screenshots/TextFilter.png"></details> | 
+[MultiSelectFilter](https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-filter-multiselectfilter--multi-select-filter) | <details><summary>Screenshot</summary><img src="src/screenshots/MultiSelectFilter.png"></details> | [RadioFilter](https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-filter-radiofilter--radio-filter) | <details><summary>Screenshot</summary><img src="src/screenshots/RadioFilter.png"></details> | | [TextFilter](https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-filter-textfilter--text-filter) | <details><summary>Screenshot</summary><img src="src/screenshots/TextFilter.png"></details> | 
```

Comment:
I think there was a miss interpretation of the issue here. The issue was not that the `SingleSelectFilter` was duplicated, but rather that the `Select` component was duplicated. Please add back all references to the `SingleSelectFilter` -- just instead of using duplicate `Select` code that existed in the SingleSelectFilter folder, utilize the official `src/Select` component.